diff_cover/tests/test_integration.py

```diff
@@ -393,6 +393,13 @@ def test_git_diff_error(self):
         with self.assertRaises(CommandError):
             diff_cover_main(["diff-cover", "coverage.xml"])
 
+    def test_quite_mode(self):
```

Comment:
This and the other test both say `quite` rather than `quiet`

Reply:
done